src/service/sharding-table-service.js

```diff
@@ -152,7 +152,6 @@ class ShardingTableService {
         firstAssertionId,
         hashFunctionId,
     ) {
-        const kbSize = assertionSize < BYTES_IN_KILOBYTE ? BYTES_IN_KILOBYTE : assertionSize;
```

Comment:
could you create a unit test for this, where all nodes have ask set to 1 wei ? I think we still set a minimum here or bid suggestion will be rounded to 0. Minimum could be 1 wei * r0 * number of epochs for example, so that we ensure all nodes will be rewarded with at least 1 wei per epoch

Reply:
yes i'm updating the tests it will be added in PR